clients/fides-js/src/lib/initialize.ts

```diff
@@ -252,18 +262,12 @@ export const initialize = async ({
       isPrivacyExperience(effectiveExperience) &&
       experienceIsValid(effectiveExperience, options)
     ) {
-      // Overwrite cookie consent with experience-based consent values
-      // eslint-disable-next-line no-param-reassign
-      cookie.consent = buildCookieConsentForExperiences(
```

Comment:
this was messing up when going from Notices --> TCF because TCF doesn't have any notices, so this was making the `consent` object `{}` which would erase if someone had made choices in CA. So this logic got moved out to only notices

Reply:
This does have an interesting side effect which I'm not sure if we can avoid. Let's say our cookie starts as
```js
{
  consent: {
    data_sales_and_sharing: false
  }
}
```
Now we switch to TCF. when TCF initializes, there are no notices, so it only has the consent defaults, which it merges with the existing cookie 

```js
{
  consent: {
    data_sales: true, 
    tracking: true, 
    analytics: true, 
    data_sales_and_sharing: true
  }
}
```
I'm not sure if it's ok to have the consent defaults hanging around like that. That's from [this function](https://github.com/ethyca/fides/blob/aking/prod-1070/tcf-geography-bundling/clients/fides-js/src/lib/cookie.ts/#L169-L175) which I think is there so that we continue support for the legacy config. We could potentially say just ignore those defaults if we are in TCF mode?

Reply:
Hmm, thinking beyond one impl on a customer's site, suppose a web user were to visit 2 different websites, site A uses Fides non-TCF and the site B uses Fides TCF. The user has submitted consent for site A, then navigates to site B. I'd think we'd want to preserve any existing consent the user has given on site A, rather than overwrite with defaults from TCF, right?

Reply:
site A and site B would use different cookies, right?

Reply:
I didn't think so. It should just be the same `fides_consent` cookie